src/main/java/io/vinyldns/java/model/batch/BatchResponse.java

```diff
@@ -103,6 +117,10 @@ public String toString() {
         + changes
         + ", status="
         + status
+        + ", ownerGroupId="
```

Comment:
We should include single quotes around `String` responses when serializing the data structures in `toString`:
```
        + ", ownerGroupId='"
        + ownerGroupId
        + '\''
        + ", ownerGroupName='"
        + ownerGroupName
        + '\''
        + '}';
```